release-notes/2024-04/Drop-workorder-database.md

```diff
@@ -0,0 +1,28 @@
+# Dropping the work order database
+
+## Assumptions
+
+- The workorder service is enabled but not in use, no data needs to be retained.
+- SystemLink Enterprise deployment uses in-cluster, per-service mongo databases.
```

Comment:
@prestwick Do we need instructions for any other cases besides per-service db instance?

Reply:
Added external